Audiobook Player/Player/PlayerViewController.swift

```diff
@@ -220,128 +209,44 @@ class PlayerViewController: UIViewController {
         self.dismiss(animated: true, completion: nil)
     }
 
-    @IBAction func setSleepTimer(_ sender: UIButton) {
-        var alertTitle: String?
-
-        if self.sleepTimer != nil && self.sleepTimer.isValid {
-            alertTitle = " "
-        }
-
-        let alert = UIAlertController(title: alertTitle, message: nil, preferredStyle: .actionSheet)
-
-        alert.addAction(UIAlertAction(title: "Off", style: .default, handler: { _ in
-            self.sleep(in: nil)
-        }))
-
-        alert.addAction(UIAlertAction(title: "In 5 Minutes", style: .default, handler: { _ in
-            self.sleep(in: 300)
-        }))
-
-        alert.addAction(UIAlertAction(title: "In 10 Minutes", style: .default, handler: { _ in
-            self.sleep(in: 600)
-        }))
+    @IBAction func setSleepTimer() {
+        let actions = SleepTimer.shared.actionSheet(
```

Comment:
This should be called something along the lines of `actionSheet` since `actions` is not an array of actions, but an instance of `UIAlertController`